dev/release/rat_exclude_files.txt

```diff
@@ -1,6 +1,10 @@
 *.json
 *.Rproj
 *.Rd
+c/subprojects/fmt*
+c/subprojects/gtest*
+c/subprojects/nanoarrow*
+c/subprojects/sqlite3*
```

Comment:
Can we set our license header to `c/subprojects/*.wrap` instead of excluding them?

Reply:
These headers were generated by `meson wrap install ...` - happy to add that if the fact that they are automatically generated is not important

Reply:
OK. Let's exclude `*.wrap`. Can we narrow exclude targets as much as possible such as using `fmt.wrap` instead of `fmt.*`?

Reply:
No problem. I made these wildcards to prevent anything generated by the subprojects themselves being included, but those should never be committed anyway. Changed to the more explicit form you have suggested